bootstrap/aws/config-default.sh

```diff
@@ -13,7 +13,8 @@ ZONE=${APOLLO_AWS_ZONE:-eu-west-1b}
 MASTER_SIZE=${MASTER_SIZE:-m1.medium}
 SLAVE_SIZE=${SLAVE_SIZE:-m1.medium}
 NUM_SLAVES=${NUM_SLAVES:-1}
-
+FRAMEWORK_MARATHON_ENABLED=${FRAMEWORK_MARATHON_ENABLED:-'Y'}
```

Comment:
For common settings / environment variables we can either use the existing ```apollo-env.sh``` (which is included everywhere) or add some other common ones that get included